R/signal.R

```diff
@@ -22,14 +22,27 @@
 #'
 #' @keywords internal
 #' @export
-signal_experimental <- function(when, what) {
-  signal(
-    "",
-    "lifecycle_experimental",
-    when = when,
-    what = what
-  )
+signal_experimental <- function(what) {
+  verbosity <- lifecycle_verbosity()
```

Comment:
I decided to just use `lifecycle_verbosity()` directly because deprecated and experimental functions are roughly equally risky.